sequence_processing_pipeline/Job.py

```diff
@@ -150,6 +150,9 @@ def _system_call(self, cmd, allow_return_codes=[], callback=None):
         :return: a dictionary containing stdout, stderr, and return_code as
                  key/value pairs.
         """
+        # when system-wide binaries such as 'sbatch' and 'sacct' are not
+        # present, assume executable w/no path is present in current
+        # directory.
```

Comment:
Is this comment still needed? It doesn't look like a change to the test runner was necessary?

Reply:
I'm going to try adding it to the environment on the qp-klp side of things. If something's needed I'll introduce it, I will.